src/hacbs/components/TaskRunListView/TaskRunListView.tsx

```diff
@@ -20,6 +36,44 @@ const TaskRunListView: React.FC<Props> = ({ pipelineName, namespace }) => {
     },
   });
 
+  const [nameFilter, setNameFilter] = React.useState<string>('');
+
+  taskRuns?.sort(
+    (a, b) => +new Date(a.metadata.creationTimestamp) - +new Date(b.metadata.creationTimestamp),
+  );
```

Comment:
Now that this page could re-render due to filtering, capture the sorted tasks inside a memo.

Reply:
updated